article/_posts/2020-05-28-NGU.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: review
+title: "Never Give Up: Learning Directed Exploration Strategies"
+tags:
+author: "Antoine Théberge"
+cite:
+    authors: "Never Give Up: Learning Directed Exploration Strategies"
+    title:   "Adrià Puigdomènech Badia, Pablo Sprechmann, Alex Vitvitskyi, Daniel Guo, Bilal Piot, Steven Kapturowski, Olivier Tieleman, Martín Arjovsky, Alexander Pritzel, Andew Bolt, Charles Blundell"
+    venue:   "ICLR2020"
+pdf: "https://arxiv.org/pdf/2002.06038.pdf"
+---
+
+
+[![https://www.youtube.com/watch?v=KxGRhd_iWuE](/article/images/NGU/ngu.jpeg)](https://www.youtube.com/watch?v=KxGRhd_iWuE)
+
+
+# Highlights
+
+- Reinforcement learning agent able to solve hard exploration games by learning a range of directed exploratory policies
+- First algorithm to achieve non-zero rewards (with a mean score of 8,400) in the game of Pitfall! without using demonstrations or hand-crafted features
+
+# Introduction
+
+> The problem of exploration remains one of the major challenges in deep reinforcement learning. 
+
+A learning agent often needs to finetune a tradeoff between exploitation (acting on what it knows will work) and exploration (behaving in a way that is not guaranteed to be good) so as to learn a policy that achieves good reward without being stuck in a local optimum.
+
+The first ideas for exploration were to have an $${\epsilon}-greedy$$ policy (taking a random action with a probability of $$1-\epsilon$$) or to do Boltzmann exploration (exploring w.r.t the Q function of the state-action pair with a temperature parameter to decrease exploration over time). However, these methods are very inefficient in the context of larger non-tabular state-spaces. 
+
+Recent approaches have proposed to equip agents with an intrinsic reward promoting exploration, making them focus on exploring new, unseen states. As the agent explores its environment, the intrinsic motivation wears off and the reward becomes extrinsic. However, once a state has been visited, the agent is no longer encouraged to visit it again and therefore might not explore subsequent states that might have been more profitable. Finetuning the "curiosity" only seems to delay the problem.
```

Comment:
The curiosity is the intrinsic reward promoting exploration, right? Maybe you can add the former between brackets when you first mention the later in the paragraph.